tests/vmi_configuration_test.go

```diff
@@ -3098,9 +3098,7 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 		var vmi *v1.VirtualMachineInstance
 
 		BeforeEach(func() {
-			if tests.IsRunningOnKindInfra() {
-				Skip("Skip KVM MSR prescence test on kind")
-			}
+			checks.SkipIfRunningOnKindInfra("Skip KVM MSR prescence test on kind")
```

Comment:
Let's make sure we follow up and remove this skip and others

Reply:
> Let's make sure we follow up and remove this skip and others

I am not sure what it means. Do you means we should remove `SkipIfRunningOnKindInfra `?

Reply:
Yes. We should not need it.

Reply:
I think we still need it in `[tests/network/networkpolicy.go](https://github.com/kubevirt/kubevirt/pull/10303/files#diff-c62c0c62d01f0c0e6234c17500859ad8f8ed735aedb34c0874374f3a38a3f96a)`

Reply:
In terms of this test, I am not farmer with KVM MSR, why we skip it in the kind env? Does it works in kind env now? I do not get much information from the message of the commit `338e7bed31`.

Reply:
Thanks @xpivarc